pkg/aws/snow.go

```diff
@@ -0,0 +1,52 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+)
+
+const (
+	snowEC2Port = 8243
+)
+
+func snowEndpoint(deviceIP string) *ServiceEndpoint {
+	return &ServiceEndpoint{
+		ServiceID:     "EC2",
+		SigningRegion: "snow",
+		URL:           fmt.Sprintf("https://%s:%d", deviceIP, snowEC2Port),
+	}
+}
+
+func WithSnow(deviceIP string, certsFile string) awsConfigOpts {
```

Comment:
nit:
```suggestion
func WithSnowEndpoint(deviceIP string, certsFile string) awsConfigOpts {
```

Reply:
it is not just endpoint, but with ca bundles and device creds